cime_config/acme/machines/template.case.run

```diff
@@ -1,416 +1,327 @@
+def runModel(case):
+###############################################################################
 
+    # Set OMP_NUM_THREADS
+    num_threads = {{ thread_count }}
+    os.environ["OMP_NUM_THREADS"] = str(num_threads)
 
+    # Run the model
+    cmd = ("sleep 10")
+    run_cmd(cmd)
+
+    logger.info("%s MODEL EXECUTION BEGINS HERE" %(time.strftime("%Y-%m-%d %H:M:S")))
+
+    # {{mpirun}} set the unresolved cmd string
+    {{ mpirun }}
+    cmd = case.get_resolved_value(cmd)
+
+    logger.debug("run command is %s " %cmd)
+    rundir = case.get_value("RUNDIR")
+    run_cmd(cmd, from_dir=rundir) 
+    logger.info( "%s MODEL EXECUTION HAS FINISHED" %(time.strftime("%Y-%m-%d %H:M:S")))
+
+###############################################################################
+def postRunCheck(case, lid):
+###############################################################################
+
+    caseroot = case.get_value("CASEROOT")
+    rundir   = case.get_value("RUNDIR")
+    model    = case.get_value("MODEL")
+
+    # find the last model.log and cpl.log
+    model_logfile = os.path.join(rundir,model + ".log." + lid)
+    cpl_logfile   = os.path.join(rundir,"cpl" + ".log." + lid)
+
+    if not os.path.isfile(model_logfile):
+        msg = "%s Model did not complete, no %s log file %s \n" %(time.strftime("%Y-%m-%d %H:%M:%S"), model, model_logfile)
+        appendCaseStatus(caseroot,msg)
+        expect(False, msg) 
+    elif not os.path.isfile(cpl_logfile):
+        msg = "%s Model did not complete, no cpl log file %s \n" %(time.strftime("%Y-%m-%d %H:%M:%S"), cpl_logfile) 
+        appendCaseStatus(caseroot, msg)
+        expect(False, msg) 
+    elif os.stat(model_logfile).st_size == 0:
+        msg = " %s Run FAILED " %time.strftime("%Y-%m-%d %H:%M:%S")
+        appendCaseStatus(caseroot, msg)
+        expect (False, msg)
+    else:
+        if 'SUCCESSFUL TERMINATION' in open(cpl_logfile).read():
+            msg = "%s run SUCCESSFUL" %time.strftime("%Y-%m-%d %H:%M:%S")
+            appendCaseStatus(caseroot, msg )
+        else:
+            msg = "%s Model did not complete - see %s \n " %(time.strftime("%Y-%m-%d %H:%M:%S"), cpl_logfile)
+            appendCaseStatus(caseroot, msg)
+            expect (False, msg)
+
+###############################################################################
+def getTimings(case, lid):
+###############################################################################
+
+    check_timing = case.get_value("CHECK_TIMING")
+    if check_timing:
+        save_timing  = case.get_value("SAVE_TIMING")
+        rundir       = case.get_value("RUNDIR")
+        caseroot     = case.get_value("CASEROOT")
+        model        = case.get_value("MODEL")
+        
+        timingDir = os.path.join(caseroot, "timing")
+        if not os.path.isdir(timingDir):
+            os.makedirs(timingDir)
+
+        logger.info("Running timing script %s " %(os.path.join(caseroot, "Tools/getTiming")))
```

Comment:
here you are sorta using os.path.join but not really ( you should never explicitly need the '/') - try:
os.path.join(caseroot, "Tools", "getTiming")